stories/Button.stories.tsx

```diff
@@ -16,6 +20,9 @@ export default {
 const Template = (args) => <Button {...args} />;
 
 export const Primary = Template.bind({});
+Primary.args = {
+  outline: false,
```

Comment:
question: reason for adding this? Outline is an optional prop with default as false so ideally we shouldn't need to set explicit arg.

Reply:
Also great question. By setting it explicitly, the addon wouldn't make combinations of it. If not, it would always do `[true, false]`. It is possible to make it work the way you mentioned, but it requires a little more logic to it:

1 - Check if the argType is of boolean type
2 - Get the default value of that given argType, if it exists
3 - If it exists, return that default value rather than make a combination out of it

However if that is the case, the addon would never provide combinations, because it would always take default values into account.